bin/millau/node/src/service.rs

```diff
@@ -161,10 +164,40 @@ pub fn new_full(config: Configuration) -> Result<TaskManager, ServiceError> {
 	let telemetry_connection_sinks = sc_service::TelemetryConnectionSinks::default();
 
 	let rpc_extensions_builder = {
+		// This struct is here to ease update process.
+
+		/// Millau runtime from message-lane RPC point of view.
+		struct MillauMessageLaneKeys;
+
+		impl pallet_message_lane_rpc::Runtime for MillauMessageLaneKeys {
+			fn message_key(&self, instance: &InstanceId, lane: &LaneId, nonce: MessageNonce) -> Option<StorageKey> {
+				match *instance {
+					RIALTO_BRIDGE_INSTANCE => Some(millau_runtime::rialto_messages::message_key(lane, nonce)),
```

Comment:
Seems that at least some of the functions inside `rialto_messages` could easily be made generic, so this implementation could be made simpler - i.e. I think the `instance` match should be made way further in the API (within the module), not here long term.
Just something to keep in mind when working on #446